compiler/luci-interpreter/src/loader/KernelBuilder.cpp

```diff
@@ -431,13 +431,6 @@ std::unique_ptr<Kernel> KernelBuilder::visit(const luci::CircleStridedSlice *nod
 {
   assert(node->arity() == 4);
```

Comment:
This also seems unnecessary.

Reply:
all circle node check arity on the begin of visit. is it okay for me to remove this?

Reply:
Sorry, I misread this. I thought it checks the rank of the shape of the input. Please ignore my comment.